tools/vol2obj/main.c

```diff
@@ -66,6 +66,7 @@
 #define STB_IMAGE_WRITE_IMPLEMENTATION
 #include "stb/stb_image_write.h"
 #include <assert.h>
+#include <stdarg.h>
```

Comment:
i was repeating code a lot so made a printf() wrapper function that could print text in different colours. hence needing `stdarg.h`